libs/pageserver_api/src/key.rs

```diff
@@ -564,35 +574,36 @@ pub const TWOPHASEDIR_KEY: Key = Key {
 };
 
 #[inline(always)]
-pub fn twophase_file_key(xid: TransactionId) -> Key {
+pub fn twophase_file_key(xid: u64) -> Key {
     Key {
         field1: 0x02,
         field2: 0,
         field3: 0,
-        field4: 0,
-        field5: 0,
-        field6: xid,
+        field4: ((xid & 0xFFFFFF0000000000) >> 40) as u32,
```

Comment:
Will it be a little bit motor efficient not to use field5 at all store epoch in field4?

Reply:
Perhaps. Seems weird to leave a hole in the representation though. And this isn't performance sensitive.